src/Sylius/Bundle/ThemeBundle/Templating/TemplateNameParser.php

```diff
@@ -74,4 +74,13 @@ public function parse($name): TemplateReferenceInterface
 
         return $this->cache[$name] = $template;
     }
+
+    private function getBundleOrPluginName(string $name): string
+    {
+        if (preg_match('/^.+Plugin$/', $name)) {
+            return $name;
+        }
+
+        return $name ? $name . 'Bundle' : '';
```

Comment:
Do we need this ternary operator here?

Reply:
Ok, in fact we need, as it's explained in `it_generates_template_references_from_root_namespaced_paths` specification